### PR TITLE
Update C++/WinRT to version 2.0.200615.7

### PR DESCRIPTION
**Pull request by vmoroz**
We currently use C++/WinRT version 2.0.200316.3. It is almost 3 months old.
This PR updates our version of C++/WinRT to the latest version 2.0.200615.7 released yesterday.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5259)

### CODE REVIEW

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.200615.7\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.200615.7\build\native\Microsoft.Windows.CppWinRT.props')" />
```
Comment (aeulitz) [inline vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:3]: ><Import Project [](start = 2, length = 15)

Not entirely sure why we need this in the DLL project.

File: vnext/Desktop/module.g.cpp [hunk 2]
```diff
@@ -30,11 +32,26 @@ void* __stdcall winrt_get_activation_factory([[maybe_unused]] std::wstring_view
         return std::equal(left.rbegin(), left.rend(), right.rbegin(), right.rend());
     };
 
+    if (requal(name, L"Microsoft.ReactNative.ReactDispatcherHelper"))
```
Comment (dannyvv) [inline vnext/Desktop/module.g.cpp:35]: Just checking if this is intentional as part of the upgrade or perhaps another change? 
Or did someone miss running the generator and checking this file in?

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj (hunk 1 above)
Comment (vmoroz) [reply vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:3]: I think DLL project must generate winmd file associated with it.

File: vnext/Desktop/module.g.cpp (hunk 2 above)
Comment (vmoroz) [reply vnext/Desktop/module.g.cpp:35]: Yes, this is an intentional change: the file generated by C++/WinRT is not compilable by C++. We must change the file and copy it locally to compile.

**Pull request merged**